packages/block-library/src/image/transforms.js

```diff
@@ -140,6 +145,29 @@ const transforms = {
 				} );
 			},
 		},
+		{
+			// When drag and dropping multiple files onto a gallery this overrrides the
+			// gallery transform in order to add new images to the gallery instead of
+			// creating a new gallery.
```

Comment:
So, it seems this transform is just generalizing the one  above it right? Can we just remove the files length check and merge them?

Reply:
Yeh that seems to work, done.